src/components/charts/Charts.tsx

```diff
@@ -17,34 +17,34 @@ export default function Charts() {
   const [yAxisSelection, setYAxis] = useState(AggregationFactor.country);
   const isMobileDeviceOrTablet = useMediaQuery({ maxWidth: mobileDeviceOrTabletWidth })
   const [state, dispatch] = useContext(AppContext);
-  const { filtered_records, filters, showAnalyzePopup } = state;
+  const { filters, showAnalyzePopup } = state;
   const initState = [] as AggregatedRecord[];
   const [showModal, toggleModal] = useState(true);
   const [records, setRecords] = useState(initState);
 
   const yAxisOptions = [
     { key: 'Geographies', text: Translate('Geographies'), value: AggregationFactor.country },
-    { key: 'Population', text: Translate('Population'), value: AggregationFactor.population_group },
+    // { key: 'Population', text: Translate('Population'), value: AggregationFactor.population_group },

```

Comment:
Can we make a backend ticket to make sure that these fixes get done. I think maybe @simonarocco can take it cause it seems to be related to meta analysis. Will add more details in the tick